### PR TITLE
add entry for 5.6.6

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Add entry for the upcoming 5.6.6 release

### CODE REVIEW

File: history.rst [hunk 1]
```diff
@@ -5,8 +5,40 @@
...
+- omero-renderer 5.5.11
+- omero-romio 5.7.1
+- omero-common 5.6.0
+- omero-model 5.6.9
+
+as well as the upgrade of omero-scripts to version 5.7.1.
+
+Improvements include:
+
+- enhancement of the Command line importer developer user experience
```
Comment (sbesson) [inline history.rst]: Possibly clarify this is the Java command line importer?

File: history.rst [hunk 2]
```diff
@@ -5,8 +5,40 @@
...
+- the option for omero.server.nodedescriptors to be queried from a client
+- a new property to configure the QA system the feedback is submitted to.
```
Comment (sbesson) [inline history.rst]: Worth naming it?

**Review by sbesson — CHANGES_REQUESTED** [review]
Two suggestions. I have not looked through the complete list of changes across acll components but I assume this focuses on the end-user facing ones

**Comment by jburel** [comment]
@sbesson adjustment made

**Review by sbesson — APPROVED** [review]

**Pull request merged**